### ISSUE TITLE
UI issues with pasting a column header named 'sample_name'

### ISSUE THREAD

**Issue by nsheff** [pepkit/pephub#342]
When I paste a column with the name 'sample_name', it removes the previous column.

To reproduce:

1. create a pep
2. paste this content: 

<style type="text/css">td{white-space:normal}br{mso-data-placement:same-cell}</style>
sample_name | sample_type | genome
-- | -- | --
sample1 | sample_type1 | genome1
sample2 | sample_type2 | genome2

3. copy the `sample_name` cell and paste it into one of the other column headers

pephub will literally delete your first column, and there is no way to get that back.

Basically, something very strange is happening; it seems like pephub is trying to be smart and auto-correct to not allow two columns named `sample_name`. But this is really problematic. I suggest we don't try to outthink the user -- just let users create the data in the table how we want, then warn us when we click save.

**Comment by nleroy917** [comment]
I think the issue here is that you **cant have duplicate column names** because of the data model we are using to store the data on the client

**Comment by nleroy917** [comment]
So, after making changes discussed in #343, you can have duplicate names on the client... but as you can see, this will lead to unexpected behavior from the server based on the *list of sample objects* data model.

![Jul-17-2024 09-46-56](https://github.com/user-attachments/assets/258fb00e-059e-4693-821b-d1be4708db26)

I suppose we could warn the user and save as usual... or warn the user and refuse to save. Not sure what is best. I think that I would prefer it doesn't save to prevent data loss on the client.

**Comment by nleroy917** [comment]
This should be fixed with new updates
